alf/algorithms/algorithm.py

```diff
@@ -308,7 +308,7 @@ def get_initial_predict_state(self, batch_size):
     def get_initial_rollout_state(self, batch_size):
         return common.zeros_from_spec(self._rollout_state_spec, batch_size)
 
-    def get_initial_train_step_state(self, batch_size):
+    def get_initial_train_state(self, batch_size):
```

Comment:
This function has the same functionality as the one from policy driver:
https://github.com/HorizonRobotics/alf/blob/0a17698f18f142711cd84560f01a9efa6d4ba052/alf/drivers/policy_driver.py#L126-L129
Maybe we can call this function in policy driver's ```get_initial_train_state```.

Reply:
policy driver will be removed.